packages/test/src/integration-tests/test-update-interceptors.ts

```diff
@@ -0,0 +1,48 @@
+import { Next, UpdateInput, WorkflowInboundCallsInterceptor } from '@temporalio/workflow';
+import { helpers, makeTestFunction } from './helpers';
+import { update, workflowWithUpdates, workflowsPath } from './workflows';
+
+const test = makeTestFunction({
+  workflowsPath,
+  workflowInterceptorModules: [require.resolve(__filename)],
+  workflowEnvironmentOpts: {
+    client: {
+      interceptors: {
+        workflow: [
+          {
+            async update(input, next): Promise<unknown> {
+              return next({ ...input, args: [input.args[0] + '-clientIntercepted', ...input.args.slice(1)] });
+            },
+          },
+        ],
+      },
+    },
+    server: {
+      executable: {
+        type: 'cached-download',
+        version: 'latest',
+      },
+    },
```

Comment:
This seems redundant to have to specify.

Reply:
I think this is the way it should be. The somewhat verbose `server` config here is temporary: the server that the tests use by default does not currently support Update.

But both `workflowEnvironmentOpts`, and the `server` property within it, are optional. I initially considered allowing the test author to just pass the `server.executable.version` string, but IMO it's cleaner to just expose established types such as `LocalTestWorkflowEnvironmentOptions`.

Reply:
> the server that the tests use by default does not currently support Update.

Can we fix this default? We control it server side for temporal.download.

Reply:
> Can we fix this default? 
I'm sure we can, but it's orthogonal to this PR. Currently the tests default to `server.executable.version=default` and this results in a server that doesn't support Update. For this PR I think it is appropriate/acceptable to set `version=latest` where needed for Update tests. As soon as `default` changes to pick up a server that supports Update (whether by a change in the semantics of that label or by server evolution) we can change it here.